src/s_audio.c

```diff
@@ -197,9 +197,10 @@ static void audio_compact_and_count_channels(int *ndev, int *devvec,
 
 /* ----------------------- public routines ----------------------- */
 
+static int initted = 0;
```

Comment:
any reason to move this out of the `sys_get_audio_settings()` body?

Reply:
that was done so that it can be seen by `sys_set_audio_settings()`

Reply:
i see (i didn't spot this due to the way GitHub displays diffs...)